client/my-sites/stats/summary/index.jsx

```diff
@@ -136,6 +137,11 @@ class StatsSummary extends Component {
 					summary />;
 				break;
 
+			case 'views':
+				title = translate( 'All Time Views' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'views all-time' )` *ES Score: 9.70*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).